README.md

```diff
@@ -0,0 +1,25 @@
+# Nookpendium
+
+-----------------------
+
+Nookpendium is an Android application developed in Kotlin with the main goal of providing Animal Crossing players a one-stop shop for all in-game resource knowledge and tracking.
+
+In it's current state, the app only supports tracking critters for Animal Crossing: New Horizons (Switch) and Animal Crossing: New Leaf, but it's planned to eventually support all Animal Crossing games with varying levels of information and tracking capabilities tailored to each version of the game.
```

Comment:
Is ACNH available for anything other than the switch? If not it may be redundant to specify that